src/igvfd/schemas/measurement_set.json

```diff
@@ -128,7 +128,7 @@
                 "Perturb-seq",
                 "CERES-seq",
                 "TAP-seq",
-                "Variant FlowFISH",
+                "Variant EFFECTS",
```

Comment:
Shouldn't it be "Variant-EFFECTS" with a hyphen?

Reply:
We didn't have a hyphen previously for "Variant-FlowFISH". I can add it here though.